webpack.config.js

```diff
@@ -160,7 +160,10 @@ const config = {
 		rules: [
 			{
 				test: /\.js$/,
-				exclude: /node_modules/,
+				exclude: [
+					/block-serialization-spec-parser/,
```

Comment:
I have no idea how it worked locally with the previous setup 🤷‍♂️ I had to exclude this package from Babel transformation.

By they way, with our current setup, it looks like we should skip transpilation for all code inside `packages` folder as soon as we fully convert `components` into a package.

Reply:
> By they way, with our current setup, it looks like we should skip transpilation for all code inside packages folder as soon as we fully convert components into a package.

Yes, this should be done at some point, the blocker is that we need to extract the i18n strings into a unique file, we do so using a babel plugin. An alternative would be to run the generation of these strings per package and concat them on the root build.